vic/vic_run/include/vic_run.h

```diff
@@ -144,13 +144,19 @@ double ErrorIcePackEnergyBalance(double Tsurf, ...);
 double ErrorPrintIcePackEnergyBalance(double, va_list);
 int ErrorPrintSnowPackEnergyBalance(double, va_list);
 int ErrorSnowPackEnergyBalance(double Tsurf, ...);
-int estimate_layer_ice_content(layer_data_struct *, double *, double *,
-                               double *, double *, double *, double *, double *,
-                               double, size_t, size_t, char);
-int estimate_layer_ice_content_quick_flux(layer_data_struct *, double *, double,
-                                          double, double, double, double *,
-                                          double *, double *, double *, double,
-                                          char);
+void estimate_frost_temperature_and_depth(double tmpT[MAX_LAYERS][MAX_NODES][MAX_FROST_AREAS + 1], double tmpZ[MAX_LAYERS][MAX_NODES], double *, double *, double *, double *, double, size_t,
```

Comment:
Break after 80 characters or run through `uncrustify` (same for the ones below here)